src/runtime/entitlements-manager.js

```diff
@@ -118,7 +117,7 @@ export class EntitlementsManager {
       this.responsePromise_ = this.getEntitlementsFlow_();
     }
     return this.responsePromise_.then(response => {
-      if (response.isReadyToPay != null) {
+      if (!(response.isReadyToPay == null)) {
```

Comment:
What's wrong with `isReadyToPlay != null`?

Reply:
while doing tests, I found that even though readyToPay expectation is not specified (null or undefined), the entitlements promise was always false. I wanted to create this check (!(response.isReadyToPay == null || response.isReadyToPay == undefined) and since they are equivalent, I changed it to !(response.isReadyToPay == null) . If this is equivalent to isReadyToPay != null then I can revert change to this line.